libs/pages/application/src/lib/ui/job-overview/job-overview.tsx

```diff
@@ -0,0 +1,93 @@
+import cronstrue from 'cronstrue'
+import { useNavigate, useParams } from 'react-router-dom'
+import { isCronJob, isLifeCycleJob } from '@qovery/shared/enums'
+import { JobApplicationEntity } from '@qovery/shared/interfaces'
+import {
+  APPLICATION_SETTINGS_CONFIGURE_URL,
+  APPLICATION_SETTINGS_RESOURCES_URL,
+  APPLICATION_SETTINGS_URL,
+  APPLICATION_URL,
+} from '@qovery/shared/router'
+import { PropertyCard } from '@qovery/shared/ui'
+
+export interface JobOverviewProps {
+  application: JobApplicationEntity
+}
+
+export function JobOverview(props: JobOverviewProps) {
+  const { application } = props
+  const { organizationId = '', environmentId = '', applicationId = '', projectId = '' } = useParams()
+  const navigate = useNavigate()
+  const path = APPLICATION_URL(organizationId, projectId, environmentId, applicationId) + APPLICATION_SETTINGS_URL
+
+  const eventsToString = (): string => {
+    if (!application) return ''
+    const events: string[] = []
+    if (application) {
+      if (application.schedule?.on_start) events.push('Start')
```

Comment:
You can't remove the `if(application) {` and add this one `if (application?.schedule?.on_start) events.push('Start')`  ?